public/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
```

Comment:
- [ ] I would like to suggest you change the default react favicon which is appearing on the tab. You can change it by just changing the `favicon.ico` by your own icon in the **public folder**.